bindings/ios/MEGASdk.h

```diff
@@ -2502,6 +2502,63 @@ typedef NS_ENUM(NSUInteger, BackupHeartbeatStatus) {
  */
 - (void)getPSA;
 
+/**
+ * @brief Get the next PSA (Public Service Announcement) that should be shown to the user
+ *
+ * After the PSA has been accepted or dismissed by the user, app should
+ * use [MEGASdk setPSAWithIdentifier:] [MEGASdk setPSAWithIdentifier:delegate:] to notify API servers about
+ * this event and do not get the same PSA again in the next call to this function.
+ *
+ * The associated request type with this request is MEGARequestTypeGetPSA.
+ *
+ * Valid data in the MEGARequest object received in onRequestFinish when the error code
+ * is MEGAErrorTypeApiOk:
+ * - [MEGARequest number] - Returns the id of the PSA (useful to call [MEGASdk setPSAWithIdentifier:]
+ *                          [MEGASdk setPSAWithIdentifier:delegate:] later)
+ * - [MEGARequest email] - Returns the URL (or an empty string)
+ * ...or for the old format:
```

Comment:
?

Reply:
| SDK  | Android |
| ------------- | ------------- |
| <img width="640" src="https://user-images.githubusercontent.com/53881615/103048053-39db2f00-45f2-11eb-8b9b-dcc43826bfce.png"> | <img width="640" src="https://user-images.githubusercontent.com/53881615/103048061-3e074c80-45f2-11eb-9171-568f1a12be6f.png"> |

As I understand from SDK and Android documentation is that the new format of the PSA return URL and the old format returns information that can be used to display inside the app.

Reply:
I mean for format of the comment is weird. why we do `...` here

Reply:
is `...or` a standard comment format?

Reply:
When `getPSA()` is called, the old format is always used.
When `getPSAwithUrl()` is called, the SDK sends to the API an additional flag `"w":1` for the command `gpsa`. The API can return still return the old format, or the new one with only a URL (see SDK-1247).

Reply:
I think @simonwng is referring the ... in the documentation:

  * - MegaRequest::getEmail - Returns the URL (or an empty string))
  * ...or for the old format:   <------ THIS LINE

Reply:
Thanks @sergiohs84 , so according to the PR https://github.com/meganz/sdk/pull/2286/files, `getPSAwithUrl` returns both URL and old format data. I would suggest removing this line of comment:
`..or for the old format:`

Reply:
Sure. Please, add a commit to update the documentation accordingly, @simonwng . Thanks

Reply:
@lancy-mega , please remove the `..or` comment from megaapi.h as well

Reply:
@lancy-mega , please remove the `..or` comment from megaapi.h as well